pyproject.toml

```diff
@@ -255,7 +255,7 @@ extend-exclude = [
 
 [tool.ruff.lint]
 extend-select = [
```

Comment:
@henryiii how did you know to remove this rule? B904? Curious as to your rationale.

Reply:
It doesn't do anything, B includes B904.

Reply:
(unlike Flake8, which needed B9)

Reply:
Gah, I misread the file and didn't see the wide selector (B). Thanks!